dipy/workflows/align.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import division, print_function, absolute_import
+import logging
+from dipy.align.reslice import reslice
+from dipy.io.image import load_nifti, save_nifti
+from dipy.workflows.workflow import Workflow
+
+
+class ResliceFlow(Workflow):
```

Comment:
For example, does this really belong in this PR? Seems unrelated to quickbundles... Maybe separate this off to another PR?